src/Stripe/Services/Subscriptions/StripeSubscriptionCreateOptions.cs

```diff
@@ -42,6 +42,24 @@ internal string TrialEndInternal
         [JsonProperty("tax_percent")]
         public decimal? TaxPercent { get; set; }
 
+        public DateTime? BillingCycleAnchor { get; set; }
+
+        public bool BillingCycleAnchorNow { get; set; }
+
+        [JsonProperty("billing_cycle_anchor")]
+        internal string BillingCycleAnchorInternal
```

Comment:
I'm not seeing this one either https://stripe.com/docs/api/curl#create_subscription. Maybe I'm looking in the wrong place?